rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/compile_time_interpreter.kt

```diff
@@ -154,6 +154,12 @@ open class BaseCompileTimeInterpreter(
     private fun findSize(statements: List<RpgParser.StatementContext>, declName: String, conf: ToAstConfiguration, innerBlock: Boolean = true): Int? {
         statements.forEach {
             when {
+                it.fspec_fixed() != null -> {
+                    val size = it.fspec_fixed().runParserRuleContext(conf) { context ->
+                        kotlin.runCatching { context.toAst(conf).let { dataDefinition -> dataDefinition to dataDefinition.toDataDefinitions() } }.getOrNull()
```

Comment:
why did you convert the instance of `FileDefinition` returned by `rpgParser.Fspec_fixedContext.toAst() `in a `Pair` when you use only e second element of this `Pair?`
It was simpler:
```kt
                it.fspec_fixed() != null -> {
                    val size = it.fspec_fixed().runParserRuleContext(conf) { context ->
                        kotlin.runCatching { context.toAst(conf).let { fileDefinition -> fileDefinition.toDataDefinitions() } }.getOrNull()
                    }?.find { dataDefinition ->  dataDefinition.name.equals(declName, ignoreCase = true) }?.elementSize()
                    if (size != null) return size
                }
```
where I have also replaced the lambda argument `it` with a more espressive name.
Remember the names are crucial for readibility and maintenance